app/controllers/user_controller.rb

```diff
@@ -537,6 +538,15 @@ def current_user_is_display_user
     @user.try(:id) == @display_user.id
   end
 
+  def show_about_me?
+    return true if @is_you
+    return false unless @display_user.get_about_me_for_html_display.present?
+    return false unless @display_user.active?
+    return true if @display_user.confirmed_not_spam?
+    return true if @user
+    false
+  end
```

Comment:
Can be simplified as:

```suggestion
  def show_about_me?
    return true if @is_you
    return false unless @display_user.get_about_me_for_html_display.present?
    return false unless @display_user.active?
    @display_user.confirmed_not_spam?
  end
```

Reply:
Or maybe a spec if missing?

Reply:
Ah needed this to check that final `return true if @user` – always want to show the `about_me` when logged in, unless there's a specific reason not to (i.e. one of the first 3 conditions).

```diff
diff --git a/spec/controllers/user_controller_spec.rb b/spec/controllers/user_controller_spec.rb
index 3e0600ded..75e6c62c9 100644
--- a/spec/controllers/user_controller_spec.rb
+++ b/spec/controllers/user_controller_spec.rb
@@ -621,7 +621,7 @@ def make_request
     end

     context 'when logged in as a different in user' do
-      before { user.update(about_me: 'x') }
+      before { user.update(about_me: 'x', confirmed_not_spam: false) }
       before { sign_in(FactoryBot.create(:user)) }
```